### ISSUE TITLE
Option to exclude files

### ISSUE THREAD

**Issue by michaeljota** [ChristianKohler/PathIntellisense#25]
Hi! Thanks for the tools. 

This is not an issue, but a suggestion. I would like to be able to exclude some files.

ex:

``` json
{
  "files.exclude": {
    "**/*.js": {"when": "$(basename).ts"},
    "**/*.map": true
  }
}
```

This would be mainly to work with ts and transpilers that generate .map also.

Again, thanks. :+1:

**Comment by ChristianKohler** [comment]
Hi @michaeljota, thank you for your feedback. I could implement a simple regex exclude filter. This would solve your problem with transpiled js files. I am not sure on the syntax yet and whether this is a feature a lot of people need since a lot of projects transpile all modules into a single file. I'll keep it open an wait for more feedback on this one.

**Comment by markfknight** [comment]
This would be mighty handy for TypeScript when developing an application as you may not have a full build / compile pipeline for development iterative steps and only rely on a full "build to single file" for deployment. Would certainly help with my work flow if the settings.json "files.exclude" could be used.

**Comment by Franziskus1988** [comment]
I'd like to see this feature, too. Especially when coding typescript only 33% of the shown files are needed.

Example where only 3 files are of interest:
![image](https://cloud.githubusercontent.com/assets/5774027/20349098/1fe63608-ac09-11e6-82f9-0cb363a0e7cd.png)

**Comment by ChristianKohler** [comment]
Fixed with version 1.3.0
